assets/css/less/colors.less

```diff
@@ -0,0 +1,7 @@
+@red: #d64226;
+@gray: #d2d2d2;
+@white: #fff;
+@brand: #6D87B9;
+@default-color: #569AF6;
+@wpbdp-primary-button-color: rgb(126,146,188);
```

Comment:
Can we make this name shorter to make it easier to use? Maybe just primary and secondary?